unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Definitions/AudioSource.proto

```diff
@@ -4,9 +4,9 @@ import "common/id.proto";
 option (ecs_component_id) = 1020;
 
 message PBAudioSource {  
-  bool playing = 1;
-  float volume = 2;
-  bool loop = 3;
-  float pitch = 4;
+  optional bool playing = 1;
+  optional float volume = 2; // default=1.0f
+  optional bool loop = 3;
```

Comment:
playing and loop doesn't have a default value specified

Reply:
Same than above